services/cd-service/pkg/service/overview.go

```diff
@@ -172,6 +173,22 @@ func (o *OverviewServiceServer) getOverview(
 							}
 						}
 					}
+					if appEnvs, err := s.GetAppsEnvs(envName, appName); err != nil {
+						return nil, err
+					} else {
```

Comment:
since you have a return in the `then` part, 
please remove the `else` to avoid unnecessary nesting